src/Microsoft.Azure.WebJobs/TableAttribute.cs

```diff
@@ -40,6 +41,11 @@ public class TableAttribute : Attribute, IConnectionProvider
         private readonly string _partitionKey;
         private readonly string _rowKey;
 
+        // Table Name Rules are summerized in this page https://msdn.microsoft.com/en-us/library/azure/dd179338.aspx
+        // Table names may contain only alphanumeric characters, can not start with a numeric character and must be 
+        // 3 to 63 characters long
+        private const string TableNameRegex = "^[A-Za-z][A-Za-z0-9]{2,62}$";
```

Comment:
Good catch. This here is the existing regex the runtime uses, so it's the definitive correct one. 
The one in Bindings.json looks like it's accounting for { } and % %tags - so it's actually incorrect.  Whereas this one is applied after the { } , %% substitutiosn. 



---
In reply to: [123387086](https://github.com/Azure/azure-webjobs-sdk/pull/1188#discussion_r123387086) [](ancestors = 123387086)